corehq/apps/commtrack/models.py

```diff
@@ -189,7 +189,7 @@ class SQLStockLevelsConfig(models.Model):
     # All of these are in months
     emergency_level = models.DecimalField(default=0.5, max_digits=4, decimal_places=2)
     understock_threshold = models.DecimalField(default=1.5, max_digits=4, decimal_places=2)
```

Comment:
should understock also be updated to 5? Seems like the two fields are related?

Reply:
I'll find out when I run the migration again (arguably I could check beforehand, just trying not to spend much time on this PR). I don't have great context on the usage of these fields, but for what it's worth, the project that triggered the overflow has an overstock threshold of 500 but an understock threshold of 10, so they don't necessarily mirror each other.